logbook-netty/src/main/java/org/zalando/logbook/netty/Conditionals.java

```diff
@@ -0,0 +1,21 @@
+package org.zalando.logbook.netty;
+
+import org.zalando.fauxpas.ThrowingConsumer;
+
+final class Conditionals {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The utility class name 'Conditionals' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper)'](https://app.codacy.com/manual/whiskeysierra/logbook/pullRequest?prid=4982439)